pkg/consts/views.go

```diff
@@ -55,6 +55,19 @@ function(doc) {
 }`,
 }
 
+// FilesByParentView is the view used for fetching files referenced by a
+// given document
+// TODO change me for flag hidden
+var FilesByParentView = &couchdb.View{
+	Name:    "by-parent-type-name",
```

Comment:
Yes, it is used for the iterator, which is used for all operation within the stack where we iterate over the children of a folder.